### DISCUSSION TITLE
Kick Off Follow Up - Action Steps

### DISCUSSION THREAD

**Discussion by ImproveAbility** [ATMakersOrg/EyeAlert#7] [Ideas]
So, Bill sprung it on me that I was to recap.  I was not taking notes throughout the call, so please correct anything here.

There are four proposed solutions that are going to be investigated further:
1. IR detection device - This would be a simple device that would detect if IR is present.  If not present, activate.  Bill has a part he is going to ship to Ron to build a prototype.
2. Project Activate - This is an existing app on the Android platform.  Antoinette, Wendy, Alisa, and Nachum will test with users and report back cases.  The idea is to try this with people while you are with them and note an issues.  Antoinette will make up a document to document feedback.
3. IR switches - Bill has worked on a previous project with IR sensors that respond to change in position.  Bill will make prototypes to be tested in the field.
4. Totally stand alone product - My understanding is a camera that would detect eye open/close state and be able to sent some sort of activation if the right criteria are met.  Bill is getting some parts to Ron to build a prototype.

**Comment by nelsonii** [comment]
For PM purposes, I'm going to be adding tickets under Issues. I work best with punch lists, and you can see where I'm at. :-)
